site-content/source/modules/ROOT/pages/development/documentation.adoc

```diff
@@ -8,18 +8,17 @@
 The official Cassandra documentation lives in the project's git
 repository.
 We use a static site generator, http://www.antora.org/[Antora], to create pages hosted at
-link:/doc/latest/[cassandra.apache.org].
-
+https://cassandra.apache.org/docs/latest/[cassandra.apache.org].
```

Comment:
This wasn't broken before, and `link:` is to be favoured. Can you revert this change please.